subcommands/waves/init.go

```diff
@@ -54,6 +61,20 @@ Example: 1,2,3`)
 	_ = initCmd.MarkFlagRequired("keys")
 }
 
+func confirmWaveInitProceed() {
+	logrus.Warning("**(Recommended)** Generate OSTree static deltas before rolling out waves to devices. " +
+		"More info: https://docs.foundries.io/latest/reference-manual/ota/static-deltas.html#ref-static-deltas \n")
```

Comment:
I am on the fence here.
Almost all our commands are non-interactive, requiring user input only in the form of arguments.
This is nice, because it works on all terminals and is easy to script.

It is totally fine to do waves without static deltas, or static deltas without waves.
We have customers doing all sort of magic using various machinery combinations.
Still, there is a definite value in showing something to the user if there are no static deltas.
So, I would prefer to do just that: suggest the user to generate static deltas (at the end of `doInitWave`), but don't interrupt the operation.
Our dialogs should be supportive, not annoying.

Also, I have suggestions to improve the message itself, subject to @kprosise remarks.

```suggestion
fmt.Println(`You created a wave for a target version without static deltas.
We recommend that you generate static deltas for your production targets to optimize the size of the update.
Please, consider generating a static delta for this target version using `fioctl targets static-deltas` command.
You can then cancel this wave and create a new one for a target with a static delta.`)
```

A rationale is:
- Put no links to documentation: they may be moved arbitrarily, while the user might use old Fioctl for a long time.
  Furthermore, the tool looks more professional when it can describe itself without a necessity to direct a user to external resource.
- This is a command-line tool, so please, use plain text: no markup, no colors, no log-styled clobbery.
  We use logs sometimes, but as intended - to show some diagnistics in debug mode; not for regular user interaction.

Reply:
@vkhoroz, looks good